shared/core/types/pools.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+var poolFriendlyNames map[Pools]string
+
+func init() {
+	poolFriendlyNames = map[Pools]string{
+		Pools_POOLS_UNSPECIFIED:        "Unspecified",
+		Pools_POOLS_DAO:                "DAO",
+		Pools_POOLS_FEE_COLLECTOR:      "FeeCollector",
```

Comment:
```suggestion
		Pools_POOLS_FEE_COLLECTOR:      "FeeCollectorPool",
```

Reply:
Are you sure you want to change this? The current values reflect what's currently on `main`, also, I could argue that they are entities in the table `pool` so the suffix is redundant.